var/spack/repos/builtin/packages/care/package.py

```diff
@@ -50,137 +67,219 @@ class Care(CMakePackage, CudaPackage, ROCmPackage):
     depends_on("cxx", type="build")  # generated
     depends_on("fortran", type="build")  # generated
 
-    variant("openmp", default=False, description="Build Shared Libs")
+    variant("openmp", default=False, description="Build with OpenMP support")
+    variant("mpi", default=False, description="Enable MPI support")
     variant(
         "implicit_conversions",
-        default=True,
+        default=False,
```

Comment:
@adayton1 I didn't even notice this change I made. We had implicit_conversions set to "False" locally. Would you prefer to have it to "True" here in Spack ?
Thanks @tldahlgren for the warning.

Reply:
I think it should default to False.